### PR TITLE
post__date: Do not draw element in the DOM-tree, if there is no data

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: src/blocks/common.blocks/post/__date/post__date.bemhtml [hunk 1]
```diff
@@ -1,48 +1,35 @@
...
-                BEM.I18N('post', 'createDate'),
-                ' ',
-                applyNext(this.ctx.content = this.ctx.createDate)
-            ]
+            return d + ' ' + BEM.I18N('post', 'month' + m) + ' ' + y;
         }
 
-        (this.ctx.createDate && this.ctx.editDate && this.ctx.editDate >= this.ctx.createDate): {
-
+        if (ctx.editDate) {
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml:21]:
``` js
return ctx.editDate ? [/* ... */] : [/* ... */];
```

File: src/blocks/common.blocks/post/_view/post_view_full.bemtree [hunk 2]
```diff
@@ -52,13 +52,10 @@ block post, mod view full {
...
-                elem: 'date',
-                elemMods: { type: 'human' },
-                createDate: createDate,
-                editDate: editDate
-            }
+        return {
+            elem: 'date',
+            createDate: createDate,
+            editDate: editDate
         }
```
Comment (tadatuta) [inline src/blocks/common.blocks/post/_view/post_view_full.bemtree]: ;

File: src/blocks/common.blocks/post/__date/post__date.bemhtml (hunk 1 above)
Comment (gela-d) [inline src/blocks/common.blocks/post/__date/post__date.bemhtml:21]: @tadatuta We discussed this case, but decided to choose this entry

**Pull request merged**